src/dndbeyond/content-scripts/character.js

```diff
@@ -848,6 +848,21 @@ async function rollItem(force_display = false, force_to_hit_only = false, force_
             }
         }
 
+        // Handle Dragon Wing * Ranged Weapons
+        if (item_name.includes("Dragon Wing") || item_name === "Flail of Tiamat") {
+            damages.splice(2,damages.length - 2);
+            let possible_damages = damage_types.splice(1,damage_types.length - 1);
+            if (properties.Notes && possible_damages.includes(properties.Notes)) {
```

Comment:
The `includes` here means it needs to match perfectly, so if there's a note like "Damage is of type fire", it won't work.
I'd rewrite this into something like this : 
```js
   const damage_type = properties.Notes && possible_damages.reduce((found, damage) => {
         if (found) return found;
         if (properties.Notes.toLowerCase().includes(damage.toLowerCase())) return damage;
         return null;
   }, null);
   if (damage_type) {
      damage_types.splice(1, 0, damage_type);
  } else if () ....
}
```